test/e2e/set/scyllacluster/scyllamanager_object_storage.go

```diff
@@ -442,7 +442,8 @@ var _ = g.Describe("Scylla Manager integration", framework.RequiresObjectStorage
 
 		verifyCQLData(ctx, di)
 	},
-		g.Entry("using default ScyllaDB version", entry{}),
+		// FIXME: Test should be enabled when 3.5.x Scylla Manager is available and bumped in CI.
+		g.PEntry("using default ScyllaDB version", entry{}),
```

Comment:
shouldn't the PR be marked as WIP?
/hold

Reply:
We need to decide one way or the other. We either proceed with having coverage for 6.0.x with 30'ish passing tests, or not having any coverage because one is failing due to known issue.

Reply:
Is it a matter of just making a manager release? I though the PR landed there or is there another fix needed?

ginkgo wise that would be a skip, instead of pending test IMO
`--fail-on-pending will fail the suite if it contains any pending specs. These are generally only used while developing the suite and should not be committed`
https://onsi.github.io/ginkgo/